resources/views/contacts.blade.php

```diff
@@ -0,0 +1,100 @@
+<x-app-layout :page-title="__('Contact Us')">
+    <div class="lg:grid grid-cols-2 gap-5">
+        <div>
+            <x-section>
+                <h2>Admins and Moderators</h2>
+
+                <p class="mb-3">
```

Comment:
Sorry, one final comment nit on this that I just noticed.

Technically this violates the HTML5 spec, as it's inserting a block-level element into a P tag. I recommend changing these wrapper `<p>` tags to divs.

ref: https://www.w3.org/TR/html401/struct/text.html#h-9.3.1
> The [P](https://www.w3.org/TR/html401/struct/text.html#edef-P) element represents a paragraph. It cannot contain [block-level elements](https://www.w3.org/TR/html401/struct/global.html#block-inline) (including [P](https://www.w3.org/TR/html401/struct/text.html#edef-P) itself).

Reply:
That was copied from `terms.blade.php`, which I used as a template.

https://github.com/RetroAchievements/RAWeb/blob/5a76467744180fc87cedc54abbb6bf04cdf858bc/resources/views/terms.blade.php#L4-L8

Reply:
Though the browser interprets it okay, It is not valid HTML and should be fixed there too. Maybe it makes sense to keep this as-is and update in a follow-up PR. I don't mind taking this on once this PR is merged 👍

Reply:
it's separate over there ;)

https://github.com/RetroAchievements/RAWeb/blob/5a76467744180fc87cedc54abbb6bf04cdf858bc/resources/views/terms.blade.php#L52-L63

Reply:
Oh, I misread the problem. I just saw "change `<p>` to `<div>`" and missed the complaint about having the list in the `<p>`.

Should the `<ul>` in terms be attributed with "list-disc list-inside" too?

Reply:
> Should the <ul> in terms be attributed with "list-disc list-inside" too?

Ah, yes, good catch. I missed that those should be displayed as lists, too.